core/src/test/java/PlayerTests.java

```diff
@@ -0,0 +1,144 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import com.mygdx.auber.Screens.PlayScreen;
+import com.mygdx.auber.entities.CrewMembers;
+import com.mygdx.auber.entities.Infiltrator;
+import com.mygdx.auber.entities.KeySystemManager;
+import com.mygdx.auber.entities.NPC;
+import com.mygdx.auber.entities.NPCCreator;
+import com.mygdx.auber.entities.Player;
+import com.mygdx.auber.Config;
+import com.mygdx.auber.Pathfinding.GraphCreator;
+import com.mygdx.auber.Pathfinding.MapGraph;
+import com.mygdx.auber.Pathfinding.Node;
+import com.mygdx.auber.Powerups.ArrestUp;
+import com.mygdx.auber.Powerups.FreezeUp;
+import com.mygdx.auber.Powerups.HighlightUp;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Colors;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.maps.tiled.TiledMap;
+import com.badlogic.gdx.maps.tiled.TiledMapTileLayer;
+import com.badlogic.gdx.maps.tiled.TmxMapLoader;
+import com.badlogic.gdx.utils.Array;
+import com.badlogic.gdx.math.Vector2;
+
+import org.junit.runner.RunWith;
+import org.mockito.internal.util.reflection.Whitebox;
+
+@RunWith(GdxTestRunner.class)
+public class PlayerTests {
+    TmxMapLoader mapLoader = new TmxMapLoader();
+    TiledMap map = mapLoader.load("assets/AuberMap.tmx");
+    Sprite sprite = new Sprite(new Texture("assets/Tutorial3.png")); // for janky reasons
+    Player player;
+    Array<TiledMapTileLayer> playerCollisionLayers = new Array<>();
+
+    KeySystemManager keySystemManager = new KeySystemManager((TiledMapTileLayer) map.getLayers().get("Systems"));
+    GraphCreator graphCreator = new GraphCreator( (TiledMapTileLayer) map.getLayers().get("Tile Layer 1"));
+
+    Infiltrator frozen_infiltrator, infiltrator_damage_system;
+    CrewMembers not_frozen_crew;
+
+    /**
+     * Tests that players can take damage
+     */
+    @Test
+    public void TakeDamageTest() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.